dali/operators/bbox/bb_flip.cc

```diff
@@ -75,22 +76,30 @@ void BbFlip<CPUBackend>::RunImpl(dali::SampleWorkspace &ws) {
   auto output_data = output.mutable_data<float>();
 
   for (int i = 0; i < input.size(); i += 4) {
-    auto bbox = ltrb_ ? BoundingBox::FromLtrb(&input_data[i], BoundingBox::NoBounds())
-                      : BoundingBox::FromXywh(&input_data[i], BoundingBox::NoBounds());
+    Box<2, float> bbox;
```

Comment:
Why not have some constructor from array `bbox(input_data)` and some method filling array as well - 
`bbox.to_array(output_data)`. Suggestion, not a must.

Reply:
I see you can use ReadBox/WriteBox.